ncm-ceph/src/main/pan/components/ceph/v2/schema-rgw.pan

```diff
@@ -0,0 +1,25 @@
+declaration template components/${project.artifactId}/v2/schema-rgw;
+
+type type_quoted_string = string with match(SELF, '^".*"$');
+
+@documentation{ configuration options for a ceph rados gateway instance }
+type ceph_rgw_config = {
+    'host' : string
+    'keyring' : string
+    'rgw_socket_path' : string = ''
+    'log_file' : absolute_file_path = '/var/log/radosgw/client.radosgw.gateway.log'
+    'rgw_frontends' : type_quoted_string = '"civetweb port=8000"' #Some bug in ceph config parsing
```

Comment:
is this bug still relevant?

Reply:
They fixed something: https://github.com/ceph/ceph/pull/6944. but quoted string is also in doc, so its a feature : http://docs.ceph.com/docs/master/start/quick-rgw/